### PR TITLE
add full webproject

### PR DESCRIPTION
**Pull request by Alfredbis29**
### in this milestone of my Project I :

- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Add my own styles to improve the look and feel.
- In the lesson about styling, I introduced several ways to add styles to a React application:
- React basic styling (inline styles, importing CSS and pre-processed CSS files).
- CSS modules.
- Styled components.
- In this project, I used React basic styling.
- For every page, I created a route.
- All 3 routes are displayed on each page.
- Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <div className="container">
+          <div className="flexin">
+            <h2 className="title">Let&apos;s do some math!</h2>
+            <Calc />
+          </div>
+        </div>
+      </div>
+    </>
+  );
+}
```
Comment (Gambit142) [inline src/Calculator.js:18]: - [x] Kindly follow the design guideline for the Calculator Page. The words _**Let's do some Maths**_ are aligned to the left side while the operating calculator is aligned to the right side. Kindly implement this change. 👍 

### Your Design

image

### Template Design

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Alfredbis29 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Used `react-router-dom` ✔️ 
- Implemented full website according to project details ✔️ 

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/Calculator.js (hunk 1 above)
Comment (Alfredbis29) [reply src/Calculator.js:18]: Hello @Gambit142 thanks so much for the professional review
according to the styles the mention this: Add my own styles to improve the look and feel.

Comment (Alfredbis29) [reply src/Calculator.js:18]: I even use react-router , you can now check well

Comment (Alfredbis29) [reply src/Calculator.js:18]: Check-in app.js

Comment (Gambit142) [reply src/Calculator.js:18]: Hi @Alfredbis29,

The instruction says you should add your style but use the wireframe provided. So you can add your styles, but please follow the wireframes provided for each page. 👍 

The only change you are to make is the one that has to do with the styling. I never said you didn't use `react-router-dom`.

Comment (Alfredbis29) [reply src/Calculator.js:18]: Okay okay , thanks so for you professional review

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Alfredbis29,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alfredbis29** [comment]
Hello @TedLivist  Thanks so much for approval my project , i appreciate your professional review

**Pull request merged**